spyder/plugins/ipythonconsole.py

```diff
@@ -1238,13 +1238,22 @@ def create_kernel_spec(self):
         ]
 
         # Environment variables that we need to pass to our sitecustomize
+        umr_namelist = CONF.get('main_interpreter', 'umr/namelist')
+
+        if PY2:
+            for umr_n in umr_namelist:
+                try:
+                    umr_n.encode('utf-8')
+                except UnicodeDecodeError:
+                    umr_namelist.remove(umr_n)
+            CONF.set('main_interpreter', 'umr/namelist', umr_namelist)

```

Comment:
Hi @ccordoba12 I am sorry I didn't see this on time, I just committed this change